apps/heft/src/pluginFramework/HeftLifecycleSession.ts

```diff
@@ -37,11 +37,9 @@ export interface IHeftLifecycleSession {
   readonly parameters: IHeftParameters;
 
   /**
-   * The cache folder for the lifecycle plugin. This folder is unique for each lifecycle plugin,
-   * and will not be cleaned when Heft is run with `--clean`. However, it will be cleaned when
-   * Heft is run with `--clean` and `--clean-cache`.
+   * The cache folder for the lifecycle plugin. This is an alias for tempFolderPath.
    *
-   * @public
+   * @deprecated Use `tempFolderPath` instead.
    */
   readonly cacheFolderPath: string;
```

Comment:
I would just straight up remove it since it's so new.

Reply:
I can't until we do another publish and another cyclic dependency bump.